wordpress-mobile/WordPress-Android
# Previews and labels for Stats widgets

This PR adds preview images and correct labels for widgets. These images/labels are visible when you try to add a widget. 

As part of this PR I'm also updating the release notes and replacing `posts` with `likes` in the Today's widget.

Design specs: 
<img width="470" alt="Screenshot 2019-06-28 at 09 53 29" src="https://user-images.githubusercontent.com/1079756/60327138-6d041a00-998b-11e9-97a4-fdcdb950fbe2.png">

Result:
<img width="358" alt="Screenshot 2019-06-28 at 09 58 49" src="https://user-images.githubusercontent.com/1079756/60327150-742b2800-998b-11e9-9245-9cb154b6dc4f.png">
<img width="354" alt="Screenshot 2019-06-28 at 09 58 34" src="https://user-images.githubusercontent.com/1079756/60327151-74c3be80-998b-11e9-975e-c5ff33873ed0.png">

To test:
* Try to add a widget
* Notice that the labels and previews are updated according to the specs
* Add Today's widget
* Notice it shows `likes` instead of `posts`

Update release notes:
